### DISCUSSION TITLE
Flax Farming while in Chia Pool

### DISCUSSION THREAD

**Discussion by JCandle** [guydavis/machinaris#241] [Q&A] [answered]
I am in a Chia Pool, specifically Flexpool. When I am farming Flax, my rewards are going to the same Flexpool Address (but for Flax). How do I fix that to send my rewards to my own wallet rather than Flexpools Flax wallet?

This is the Flax address that has a bunch of Flax from Flexpool Chia folks.

https://flaxexplorer.org/blockchain/address/xfx1d9rdnrj4xemphudpxrh7grs9qdtmtgnz9cn6znh9n4whv3relf2sjzg9hf

**Comment by JCandle** [comment]
For what it is worth, my target address in my farming config for Flax is my own personal one.

**Comment by guydavis** [comment, marked as answer]
Hi, I don't think that Flax yet supports official Chia pooling to a pool such as Flexpool.  They just [released an updated Flax](https://github.com/Flax-Network/flax-blockchain/releases/tag/0.1.1) that could handle the new portable plots.  However they are still working on pooling support to my knowledge.  As soon as the update is available we'll adopt it in Machinaris.  Hope this helps!
